doc/manual/src/installation/env-variables.md

```diff
@@ -62,3 +61,22 @@ If any of these variables are set when running the Nix installer, then
 the installer will create an override file at
 `/etc/systemd/system/nix-daemon.service.d/override.conf` so `nix-daemon`
 will use them.
+
+## XDG Base Directory
+
+New Nix commands conform to the [XDG Base Directory] standard,
+and uses the following environment variables to determine locations of various
+state and configuration files:
+
+- `$XDG_STATE_HOME` (assumed to be `~/.local/share` if not set): used
+  for storing state, such as the current user environment and
+  channels, repl history, and allowed flake settings.
+- `$XDG_CACHE_HOME` (assumed to be `~/.cache` if not set): used for
+  storing cache, such as flake evaluation cache and eval-time fetcher
+  results.
+
+Legacy Nix commands can also be made to follow this standard using the `use-
+xdg-base-directories` config option. However, this might cause breakage in tools
+which rely on old, non-XDG-compliant locations.
+
+[xdg base directory]: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
```

Comment:
In the linked docs, and other places online where the term is used, I see "XDG Base Directory".

Is that preferable to all lowercase?